features/visitor_can_sign_up.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Visitor can sign up
+	As a user,
+	In order to finalise an order
+	I would like to be able to sign up
+
+	Background:
+		Given I visit the landing page
```

Comment:
indention is off in this file, please follow your own standard in the different files.